src/agent/application.cpp

```diff
@@ -121,8 +121,13 @@ otbrError Application::Run(void)
         // See https://www.freedesktop.org/software/systemd/man/sd_notify.html
         sd_notify(0, "READY=1");
     }
+#endif
+
+#if defined(HAVE_LIBSYSTEMD) && OTBR_ENABLE_NOTIFY_UPSTART
     else
 #endif
+
```

Comment:
Perhaps, we must remove this line.
```suggestion
```

Reply:
shouldn't we use `else` when both HAVE_LIBSYSTEMD and OTBR_ENABLE_NOTIFY_UPSTART is ON ?